king_phisher/client/gui_utilities.py

```diff
@@ -102,6 +102,29 @@ def which_glade():
 	"""
 	return find.data_file(os.environ.get('KING_PHISHER_GLADE_FILE', 'king-phisher-client.ui'))
 
+def _store_extend(store, things, clear=False):
+	if clear:
+		store.clear()
+	for thing in things:
+		store.append(thing)
+
+def glib_idle_add_store_extend(store, things, clear=False, wait=False):
```

Comment:
Can you add this function to `docs/source/king_phisher/client/gui_utilities.rst` so the docs include it?